### DISCUSSION TITLE
Better Documentation Training Options

### DISCUSSION THREAD

**Discussion by dorofino** [NVlabs/instant-ngp#811] [General]
We need better documentation regarding all training options.
Basically, we need to understand when is good to use each of the setting to product a better NeRF

- Train Encoding
- Train Network
- Random Levels
- Train Envmap
- Train Extrinsics
- Train Exposure
- Train distortion

Best
Diego

**Comment by stevekm** [comment]
Agreed that the docs could use a general over haul. Especially for install guidelines
